api/src/main/java/org/pmiops/workbench/rdr/RdrExportService.java

```diff
@@ -10,7 +10,7 @@
 
   void exportUsers(List<Long> usersToExport);
 
-  void exportWorkspaces(List<Long> workspacesToExport);
+  void exportWorkspaces(List<Long> workspacesToExport, Boolean backfill);
```

Comment:
not addressed

Reply:
I know we chatted about this on, but I still feel like I can go either way on this one.  Passing through a backfill of null and  not sending the query param to the RDR server seems reasonable as well. If they changed the default behavior that would seem like a breaking change. I would like to leave this as-is.

Reply:
To capture the offline discussion from last week for future reference, my rationale on this was as follows:

1. Simply passing through the incoming `backfill` value creates an unnecessary implicit dependency between Workbench and RDR that they both interpret "absence of ?backfill" to mean "?backfill=false". This is because Workbench is not simply passing through the value, it's also changing its own behavior based on the interpretation of ?backfill=false.
2. In isolation, it is more natural for a Java service to accept a primitive boolean. The fact that under the hood this turns into a `Boolean` is an implementation detail.

That said, as I mentioned before this is a very minor semantic point and there were some more nuanced considerations than my first comment made out to be (which also, should have been phrased better). It's definitely not blocking.